util/Migrator/DbScripts/2023-11-27_00_2019FamilyPlanPremium.sql

```diff
@@ -0,0 +1,372 @@
+BEGIN TRANSACTION;
```

Comment:
@rkac-bw unnecessary per past conversations right? Or is something different here?

Reply:
there are two transactions but one is into temp, so seems unnecessary to have a transaction wrapping both